benefits/core/migrations/local_fixtures.json

```diff
@@ -324,18 +324,12 @@
     "fields": {
       "name": "(MST) test payment processor",
       "api_base_url": "http://server:8000",
-      "api_access_token_endpoint": "access-token",
-      "api_access_token_request_key": "request_access",
-      "api_access_token_request_val": "REQUEST_ACCESS",
+      "client_id": "",
+      "client_secret_name": "",
+      "audience": "",
       "card_tokenize_url": "http://server:8000/static/tokenize.js",
       "card_tokenize_func": "tokenize",
-      "card_tokenize_env": "test",
-      "client_cert": 5,
```

Comment:
Let's go ahead and delete the sample fixtures for the (now) unused `PemData` references as well.

Reply:
Done in [ed26642](https://github.com/cal-itp/benefits/pull/1905/commits/ed266425587aee8d315eca23ca20d2961d7bbbc3)